openr/spark/tests/Spark2Test.cpp

```diff
@@ -0,0 +1,1402 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+#include <chrono>
+#include <forward_list>
+#include <mutex>
+#include <thread>
+
+#include <folly/MapUtil.h>
+#include <folly/Memory.h>
+#include <folly/Optional.h>
+#include <folly/Random.h>
+#include <folly/ScopeGuard.h>
+#include <folly/ThreadLocal.h>
+#include <folly/system/ThreadName.h>
+#include <glog/logging.h>
+#include <gtest/gtest.h>
+#include <sodium.h>
+#include <thrift/lib/cpp2/protocol/Serializer.h>
+
+#include <openr/common/Constants.h>
+#include <openr/common/NetworkUtil.h>
+#include <openr/common/Util.h>
+#include <openr/spark/IoProvider.h>
+#include <openr/spark/SparkWrapper.h>
+#include <openr/spark/tests/MockIoProvider.h>
```

Comment:
What u need is just a different include dir of MockIoProvider.

#include <openr/tests/mocks/MockIoProvider.h>